tools/infer/predict_det.py

```diff
@@ -383,10 +381,18 @@ def __call__(self, img):
 if __name__ == "__main__":
     args = utility.parse_args()
     image_file_list = get_image_file_list(args.image_dir)
-    text_detector = TextDetector(args)
     total_time = 0
     draw_img_save_dir = args.draw_img_save_dir
     os.makedirs(draw_img_save_dir, exist_ok=True)
+    
+    # logger
+    log_file = args.save_log_path
+    if os.path.isdir(args.save_log_path) or (not os.path.exists(args.save_log_path) and args.save_log_path.endswith("/")):
+        log_file = os.path.join(log_file, "benchmark_detection.log")
+    logger = get_logger(log_file=log_file)
```

Comment:
seems this logger should be prepared and passed into AutoLog as a arg in https://github.com/PaddlePaddle/PaddleOCR/blob/main/tools/infer/predict_det.py#L168

I am worrying about maintenance of AutoLog module, it's been implemented in https://github.com/LDOUBLEV/AutoLog, and published to a server as documented here: https://github.com/PaddlePaddle/PaddleOCR/blob/38c0c9ee7715d0600c0c7ef2784b3ce64edd0710/test_tipc/prepare.sh#L204.

I think this is only been used by a terminated [tipc plan](https://github.com/PaddlePaddle/PaddleOCR/tree/main/test_tipc), we should redesign this feature, or deprecate this feature.

Reply:
> I am worrying about maintenance of AutoLog module, it's been implemented in https://github.com/LDOUBLEV/AutoLog, and published to a server as documented here:

In this case PaddleOCR could implement its own logger, instead of using AutoLog.

I think that it's a little bit of a pity to have a feature to run benchmarks without the option to save the results to a file (and only seeing them on screen instead), therefore I think this feature would be very useful.

Reply:
agree